### DISCUSSION TITLE
Hosted LLM

### DISCUSSION THREAD

**Discussion by EduardDoronin** [mckaywrigley/chatbot-ui#1517] [Help]
I am hosting ChatbotUI on a Ubuntu Linux system and am trying to use the models that Ive installed on there from the outside but cant quite figure out how I should do that. Any ideas?

**Comment by EduardDoronin** [comment]
PS.: I can only use the models on the ubuntu linux system but not from anywhere else.

**Comment by EduardDoronin** [comment]
bump :p

**Reply by wrapss** [reply]
do you use ollama for your models?

**Reply by EduardDoronin** [reply]
yes, we fixed it though, turns out the routing on the nginx server was messed up

**Reply by MuhammedAYBU2020** [reply]
HELLO. I cannot communicate with local ollama models. The system gives a 404 response. How can I solve this problem?
